how-to-run-a-node/nodes.md

```diff
@@ -2,28 +2,37 @@
 description: RESTful API to access the VeChainThor blockchain
 ---
 
-# Nodes
+# VeChainThor Nodes: Your Gateway to Blockchain Interaction
 
 ## Overview
 
-VeChain nodes are the gateway to interacting with the VeChainThor blockchain. The VeChainThor blockchain uses a RESTful application programmable interface (API) communication approach, this is referred to as the Thorest API.
+VeChain nodes serve as the critical access points to the VeChainThor blockchain, employing a RESTful API communication approach known as the Thorest API. This modern interface facilitates seamless interaction with the blockchain, enabling developers and users to harness its full potential.
 
-VeChain and several community projects provide access to public mainnet and testnet VeChainThor nodes.
+## Thorest API: Powering Blockchain Communication
 
-## Thorest API
-
-The Thorest API is the API of the VeChainThor blockchain. It is the main way to communicate and interact with the VeChainThor blockchain.
+The Thorest API stands as the primary communication channel for the VeChainThor blockchain. It offers a comprehensive set of endpoints that allow for diverse interactions, from querying blockchain data to submitting transactions.
 
 For the latest information of each of the available endpoints please use the following link [Thorest API Repo](https://github.com/vechain/thor/blob/16c5d34cfea8262e9fe91bd33b298c8e2f81da21/api/doc/thor.yaml).
 
-## Node Status Page
+## Monitoring Node Health
 
-VeChain Energy also provide a useful VeChain status page to check the health of the nodes [https://nodes.status.vechain.energy/](https://nodes.status.vechain.energy/).
+To ensure optimal performance and reliability, VeChain Energy offers a dedicated status page for node health monitoring. Check it at [https://nodes.status.vechain.energy/](https://nodes.status.vechain.energy/).
```

Comment:
There is also our own status page if you are interested. 
https://status.vechain.org

Filters:
https://mainnet.status.vechain.org
https://testnet.status.vechain.org

Reply:
Added, thanks!